src/models/room.js

```diff
@@ -38,7 +38,7 @@ import ReEmitter from '../ReEmitter';
 // to upgrade (ie: "stable"). Eventually, we should remove these when all homeservers
 // return an m.room_versions capability.
 const KNOWN_SAFE_ROOM_VERSION = '1';
-const SAFE_ROOM_VERSIONS = ['1', '2'];
+const SAFE_ROOM_VERSIONS = ['1', '2', '3'];
```

Comment:
Shouldn't this change be made in a separate PR? Normally I wouldn't mind personally, but this seems like the kind of change that should be done by itself.

Reply:
backed out to https://github.com/matrix-org/matrix-js-sdk/pull/893